sbt scrooge fix

heligw opened the pull request:
testing scrooge sbt fix

heligw committed:
sbt scrooge fix

heligw committed:
sbt scrooge fix

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/twitter/scrooge?pullRequest=355) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/twitter/scrooge?pullRequest=355) before we can accept your contribution.<br/>**2** out of **5** committers have signed the CLA.<br/><br/>:white_check_mark: vkostyukov<br/>:white_check_mark: steveniemitz<br/>:x: heligw<br/>:x: jyanJing<br/>:x: bryce-anderson<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/twitter/scrooge?pullRequest=355) it.</sub>

heligw committed:
add imports

heligw committed:
add imports

vkostyukov committed:
Revert "Twitter-oss: update OSS libraries post-release to 22.10.0-SNAPSHOT"

This reverts commit 7915f823ec292ddd448489b79935691dcad756bf.

Differential Revision: https://phabricator.twitter.biz/D943947

Commit:
Downgrade Scoverage plugin version

Problem

There is a conflict between scalatest and scoverage with the current
version combination that is causing some of the builds to fail.

Solution

Let's just downgrade scoverage until we get a better solution.

JIRA Issues: CSL-12207

Differential Revision: https://phabricator.twitter.biz/D943932

heligw committed:
add missing imports

heligw committed:
add missing imports

vkostyukov committed:
Twitter-oss: update OSS libraries post-release to 22.10.0-SNAPSHOT (revert of a revert)

This reverts commit 04ecaac33a1bd2ddb9eede8e8264ede5af3fd161.

Differential Revision: https://phabricator.twitter.biz/D944382

heligw committed:
Merge branch 'sbt/scrooge/test' of github.com:twitter/scrooge into sbt/scrooge/test

jyanJing committed:
scrooge-generator: Move `ServerValidationMixin` trait to be inside of the service companion object

Problem
During service inheritance, when parent service has validation
annotations, both parent and child services will generate a
`ServerValidationMixin` trait and cases ambiguities when
calling the trait from child service.

Solution
Move `ServerValidationMixin` to be inside of the companion
object of the service.

JIRA Issues: CSL-12205

Differential Revision: https://phabricator.twitter.biz/D943975

Commit:
Remove scoverage integration

Problem

We've run into more problems with scoverage integration.

Solution

Remove scoverage. It's a cool feature but effectively never utilized
anyway, resulting in all pain no gain.

JIRA Issues: CSL-12211

Differential Revision: https://phabricator.twitter.biz/D945589

Commit:
scrooge: Remove noisy validation warnings

Problem

Scrooge logs a large volume of warnings around validation that are not actionable.

Solution

Remove the logs

Differential Revision: https://phabricator.twitter.biz/D944195
